modules/app/controllers/webhook.js

```diff
@@ -39,6 +39,26 @@ exports.github = async (req, res) => {
   const response = req.body || res.body
   const labels = response && response.issue && response.issue.labels
   if (req.headers.authorization === `Bearer ${process.env.GITHUB_WEBHOOK_APP_TOKEN}`) {
+    // below would update issue status if someone updates it on Github
+    if (response.action === 'reopened' || response.action === 'opened' || response.action === 'closed') {
+      const status = response.issue.state
+      const dbUrl = response.issue.html_url
+      const updated = await models.Task.update({ status: status }, {
+        where: {
+          url: dbUrl
+        }
+      })
```

Comment:
There's a way to return the updated model like here:
https://github.com/worknenjoy/gitpay/blob/4df8fb6710f5d471e033de4d31dc581a99817b03/modules/app/controllers/webhook.js#L223

Reply:
But I am trying a logic similar to this.
>https://github.com/worknenjoy/gitpay/blob/4df8fb6710f5d471e033de4d31dc581a99817b03/modules/app/controllers/webhook.js#L86

Reply:
Yeah @ShubhamCanMakeCommit , but in this case, we just want to check if it was updated and in that case, we need to return the object, so it would be more convenient to return the object updated. Yeah, it will return one array with the first index to return if it was updated and index 1 to data itself.